bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -365,6 +382,31 @@ function findOutputApksHelper (dir, build_type, arch) {
     return ret;
 }
 
+function findOutputBundlesHelper (dir, build_type) {
+    var shellSilent = shell.config.silent;
```

Comment:
Hmm, majority of this method was copied from `findOutputApksHelper`, but they don't reset `shell.config.silent` either. Instead they do `shellSilent = shellSilent;` which I think is also a bug... It definitely looks like it is meant to be reset before returning.

Reply:
I don't want to make changes without fully understanding the consequences. `findOutputBundlesHelper` mimics the existing method `findOutputApksHelper`.

Because `shell` is going to be removed soon and replaced with `fs-extra` I won't be making any changes relating to shell. I will however remove the usages of `var` and replace them with `const` or `let` as appropriate.

Reply:
Thanks @breautek.

> […]
> `findOutputBundlesHelper` mimics the existing method `findOutputApksHelper`.

I wonder if we (you) should add this information to a comment?

> Because `shell` is going to be removed soon and replaced with `fs-extra`
> […]

Whoever replaces `shelljs` with `fs-extra` may want to factor out some duplicated code at the same time, if possible.

Reply:
I could write a comment stating that with the `fs-extra` refactor it may be a good idea to try to generalise the "findOutput" methods..